main/src/cgeo/geocaching/export/GpxExport.java

```diff
@@ -96,11 +96,15 @@ private Dialog getExportDialog(final String[] geocodes, final Activity activity)
         final CheckBox shareOption = ButterKnife.findById(layout, R.id.share);
         shareOption.setChecked(Settings.getShareAfterExport());
 
+        final CheckBox includeFoundStatus = ButterKnife.findById(layout, R.id.include_found_status);
+        includeFoundStatus.setChecked(Settings.getIncludeFoundStatus());
```

Comment:
Note: check if this is needed or if the default state can be set in the layout itelf (same for "share").